bundle/edu.gemini.itc.shared/src/main/scala/edu/gemini/itc/shared/InstrumentDetails.scala

```diff
@@ -82,3 +82,28 @@ final case class GemsParameters(
                      avgStrehl:           Double,
                      strehlBand:          String)
 
+
+object InstrumentDetails {
+
+  // NOTE: This is similar to the code in ItcUniqueConfig which decides on imaging or spectroscopy setup
+  // on Config elements. There should be a way to share this?
```

Comment:
(I should probably mention that the usage of Strings and the definition of `MichelleParameters`and `TrecsParameters` as Java classes is historical, this is how all these objects were originally implemented, I've changed this instrument by instrument, now only Michelle and TReCS are left; the idea is to turn them into typed `InstrumentDetail` case classes.)